src/frontend/magnify/src/components/rooms/InstantRoom/InstantRoom.tsx

```diff
@@ -0,0 +1,73 @@
+import { Form, Formik } from 'formik';
+import { Box, Button } from 'grommet';
+import * as React from 'react';
+import { defineMessages } from 'react-intl';
+import { useRouting } from '../../../context';
+import { useTranslations } from '../../../i18n';
+import FormikInput from '../../design-system/Formik/Input';
+
+const messages = defineMessages({
+  startButtonLabel: {
+    id: 'components.rooms.instantRoom.startButtonLabel',
+    defaultMessage: 'Start',
+    description: 'Label for the button to start an instant room',
+  },
+  startInputPlaceHolder: {
+    id: 'components.rooms.instantRoom.startInputPlaceHolder',
+    defaultMessage: 'Room name',
+    description: 'Label for the input to start an instant room',
+  },
+});
+
+type FormValues = {
+  roomName: string;
+};
+
+const initialValues: FormValues = {
+  roomName: '',
+};
+
+export const InstantRoom = () => {
+  const routing = useRouting();
+  const intl = useTranslations();
+
+  const randomRoom = (length: number = 40): string => {
+    let result = '';
+    const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ_abcdefghijklmnopqrstuvwxyz-0123456789';
+    let charactersLength = characters.length;
+    for (let i = 0; i < length; i++) {
+      result += characters.charAt(Math.floor(Math.random() * charactersLength));
+    }
+    return result;
+  };
```

Comment:
Just an idea for later. Instead of generating a random string hard to remember we could generate random human readable room name like it did in jitsi-box (https://github.com/openfun/jitsi-box/blob/main/box-ui/src/dictionnaries_fr/index.ts)

Reply:
Agreed !